scheds/rust/scx_lavd/src/bpf/util.bpf.c

```diff
@@ -227,6 +227,25 @@ static bool is_migration_disabled(const struct task_struct *p)
 	return false;
 }
 
+static __always_inline
+const struct cpumask *get_idle_cpumask(void)
+{
+	if (is_smt_active)
+		return scx_bpf_get_idle_smtmask();
```

Comment:
Hm... but in this way when `is_smt_active == true` we'll only consider full-idle cores and never allocate idle CPUs with a thread running. Is this intended? Maybe we want to check the idle smtmask and, if all the cores are busy, use idle cpumask?